README.md

```diff
@@ -66,19 +66,25 @@ Gear node can run in a single Dev Net mode or you can create a Multi-Node local
     - **Linux x64**: [gear-nightly-x86_64-unknown-linux-gnu.tar.xz](https://get.gear.rs/gear-nightly-x86_64-unknown-linux-gnu.tar.xz)
     - **Windows x64**: [gear-nightly-x86_64-pc-windows-msvc.zip](https://get.gear.rs/gear-nightly-x86_64-pc-windows-msvc.zip)
 
-2. Run Gear node without special arguments to get a node connected to the testnet:
+2. Run Gear node without special arguments to get a node connected to Vara Network:
 
     ```bash
     gear
```

Comment:
Current default behavior is connection to testnet. To connect to mainnet ./gear --chain vara